packages/core/contracts/common/interfaces/IERC20Standard.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+
+/**
+ * @title ERC20 interface that includes the decimals read only method.
+ */
+interface IERC20Standard is IERC20 {
+    /**
+     * @dev Returns the number of decimals used to get its user representation.
+     * For example, if `decimals` equals `2`, a balance of `505` tokens should
+     * be displayed to a user as `5,05` (`505 / 10 ** 2`).
+     *
+     * Tokens usually opt for a value of 18, imitating the relationship between
+     * Ether and Wei. This is the value {ERC20} uses, unless {_setupDecimals} is
+     * called.
+     *
+     * NOTE: This information is only used for _display_ purposes: it in
+     * no way affects any of the arithmetic of the contract, including
+     * {IERC20-balanceOf} and {IERC20-transfer}.
+     */
+    function decimals() external view returns (uint8);
```

Comment:
This is exposed to `PerpetualCreator` so that it can create token and collateral currency with the same precision